templates/cluster-template-aks.yaml

```diff
@@ -65,6 +65,7 @@ metadata:
   name: agentpool0
   namespace: default
 spec:
+  maxPods: 24
```

Comment:
does this need to be re-generated?

Reply:
I don't think so (correct me if I'm wrong), these changes are the result of running `make generate-flavors` after updating `templates/flavors/aks/cluster-template.yaml`.